snuba/datasets/entities/events.py

```diff
@@ -139,8 +142,11 @@ def v2_selector_function(query: Query, referrer: str) -> Tuple[str, List[str]]:
     ):
         return ("errors_v1", [])
 
-    range = get_time_range(query, "timestamp")
-    if range[0] is None or range[0] < settings.ERRORS_UPGRADE_BEGINING_OF_TIME:
+    time_range = get_time_range(query, "timestamp")
+    if (
+        time_range[0] is None
+        or time_range[0] < settings.ERRORS_UPGRADE_BEGINING_OF_TIME
+    ):
         return ("errors_v1", [])
```

Comment:
Not an aspect of this change, but could we describe the semantic meaning of what v2_selector function is returning somewhere?

Given the vagueness of str, List[str] I wonder if these values could be represented as enums

Reply:
We can make the improvement you are suggesting. But I would prefer to do that in a separate PR since it would touch more files.